core/api/src/servers/graphql-main-server.ts

```diff
@@ -61,15 +60,6 @@ const setGqlContext = async (
     ip,
   })
 
-  if (gqlContext instanceof Error) {
-    recordExceptionInCurrentSpan({
-      error: gqlContext,
-      fallbackMsg: "error executing sessionPublicContext",
-    })
-    next(gqlContext)
```

Comment:
the key is to not use this?

Reply:
yes, this throw an error and we dont have a global express error handler. the change "hide" the not found error and record it as a critical in the span